### PR TITLE
fix (db): Thrash

### PR DESCRIPTION
**Pull request by acidmanifesto**
Thrash was messed up extremely with a ton of npcs, this is a bit of a revert dealing with those npcs but with corrections to ensure the spell is not spammed and triggers only when they are suppose to.

## Changes Proposed:
-  Fix thrash on affected NPCS

## Issues Addressed:

- Closes I am pretty sure there is more then a few issues mentioned

## Tests Performed:

- Builds and Performs
- Affected rows: 318  Found rows: 0  Warnings: 0  Duration for 157 queries: 0.140 sec. 

## How to Test the Changes:

1. Go piss off a npc with thrash
2. Notice how they are not Excessively Over Power on their damange
3. Notice how they no longer have Excessively huge range of physical attack
4. May be better if you root then and step away when they have thrash to ensure the physical damage range is corrected

## Known Issues and TODO List:

- [x] Do alot of sugar booger after this fubar clean up
- [x] Pray to Cthulhu for the world to end

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1646161855065886626.sql [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+
+-- Skullsplitter Hunter
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 669);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(669, 0, 0, 0, 4, 0, 100, 1, 0, 0, 0, 0, 0, 11, 3621, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Skullsplitter Hunter - On Aggro - Cast \'3621\' (No Repeat)'),
+(669, 0, 1, 0, 25, 0, 100, 0, 0, 0, 0, 0, 0, 11, 12787, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Skullsplitter Hunter - In Combat - Cast \'Thrash\' (No Repeat)'),
+(669, 0, 2, 0, 2, 0, 100, 1, 5, 25, 0, 0, 0, 11, 3148, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Skullsplitter Hunter - Between 5-25% Health - Cast \'3148\' (No Repeat)');
+
+-- Tunnel Rat Kobold
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 1202));
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1646161855065886626.sql:16]: Double parenthesis?

File: data/sql/updates/pending_db_world/rev_1646161855065886626.sql [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+(2681, 0, 0, 0, 25, 0, 100, 0, 0, 0, 0, 0, 0, 11, 12787, 2, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Vilebranch Raiding Wolf  - In Combat - Cast \'Thrash\'');
+
+-- Dustbelcher Mauler
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 2717);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2717, 0, 0, 0, 4, 0, 10, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Dustbelcher Mauler - On Aggro - Say Line 0'),
+(2717, 0, 1, 0, 25, 0, 100, 0, 0, 0, 0, 0, 0, 11, 8876, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Dustbelcher Mauler - In Combat - Cast \'Thrash\'');
+
+-- Feral Crag Coyote
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 2728));
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1646161855065886626.sql:60]: Same here, 2 closing parentheses

File: data/sql/updates/pending_db_world/rev_1646161855065886626.sql [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 9096);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(9096, 0, 0, 0, 0, 0, 100, 2, 4000, 13200, 6600, 14400, 0, 11, 15580, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Rage Talon Dragonspawn - In Combat - Cast \'Strike\' (Phase 1) (No Repeat) (Normal Dungeon)'),
+(9096, 0, 1, 0, 0, 0, 100, 2, 7700, 17100, 20300, 34200, 0, 11, 12021, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Rage Talon Dragonspawn - In Combat - Cast \'Fixate\' (Phase 1) (No Repeat) (Normal Dungeon)'),
+(9096, 0, 2, 0, 0, 0, 100, 2, 1400, 12300, 7100, 11700, 0, 11, 15572, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Rage Talon Dragonspawn - In Combat - Cast \'Sunder Armor\' (Phase 1) (No Repeat) (Normal Dungeon)')
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1646161855065886626.sql:256]:
```suggestion
(9096, 0, 2, 0, 0, 0, 100, 2, 1400, 12300, 7100, 11700, 0, 11, 15572, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Rage Talon Dragonspawn - In Combat - Cast \'Sunder Armor\' (Phase 1) (No Repeat) (Normal Dungeon)'),
```

File: data/sql/updates/pending_db_world/rev_1646161855065886626.sql [hunk 4]
```diff
@@ -0,0 +1,461 @@
...
+(22174, 0, 0, 0, 25, 0, 100, 0, 0, 0, 0, 0, 0, 11, 3391, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Trigul - In Combat - Cast \'3391\''),
+(22174, 0, 1, 0, 2, 0, 100, 1, 20, 80, 0, 0, 0, 11, 33628, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Trigul - Between 20-80% Health - Cast \'33628\' (No Repeat)');
+
+-- Outraged Raven's Wood Sapling
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 21040);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(21040, 0, 0, 0, 2, 0, 100, 1, 20, 80, 0, 0, 0, 11, 3391, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Outraged Raven\'s Wood Sapling - Between 20-80% Health - Cast \'3391\' (No Repeat)');
+
+-- Fel Rager
+DELETE FROM `smart_scripts` WHERE;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1646161855065886626.sql:438]:
```suggestion
DELETE FROM `smart_scripts` WHERE  `entryorguid` = 22286;
```

File: data/sql/updates/pending_db_world/rev_1646161855065886626.sql [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+(1202, 0, 0, 0, 4, 0, 10, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Tunnel Rat Kobold - On Aggro - Say Line 0'),
+(1202, 0, 1, 0, 25, 0, 100, 0, 0, 0, 0, 0, 0, 11, 8876, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Tunnel Rat Kobold - In Combat - Cast \'Thrash\' (No Repeat)'),
+(1202, 0, 2, 0, 2, 0, 100, 1, 0, 15, 0, 0, 0, 25, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Tunnel Rat Kobold - Between 0-15% Health - Flee For Assist'),
+(1202, 0, 3, 0, 4, 0, 100, 1, 0, 0, 0, 0, 0, 1, 2, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Tunnel Rat Kobold - On Aggro - Say Line 2');
+
+-- Skeletal Flayer
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 1783;
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 1783) AND (`source_type` = 0) AND (`id` IN (0));
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1646161855065886626.sql:26]:
```suggestion
DELETE FROM `smart_scripts` WHERE (`entryorguid` = 1783);
```

**Comment by Nefertumm** [comment]
Finally fixed all the things :D

**Comment by acidmanifesto** [comment]
Now on to what really matters! Sugar Booger!

**Comment by acidmanifesto** [comment]
Fyi. Ill handle the dry run error when i get home.

**Review by Nefertumm — APPROVED** [review]

**Comment by Lynethris** [comment]
Testing in progress. I will edit this post during my progression : 

- [ ] Riverpaw Outrunner (in progress)
- [ ] Skullsplitter Hunter (in progress)
- [ ] Tunnel Rat Kobold (in progress)

**Pull request merged**